### DISCUSSION TITLE
Temperatur Sensor Values nur in 0,5 Grad Schritten

### DISCUSSION THREAD

**Discussion by JohnWayne007** [jensweigele/ioBroker.yahka#322] [Q&A] [answered]
Ich habe einen KNX Temperatursensor, die Werte landen im IO Broker als einstellige Kommawerte.
In der Ausgab auf der Home App werden sie auf ganze und halbe Grad gerundet. Also echter Wert z.b. 1,6 --> in der Home App angezeigter Wert 1,5

Egal was in in yahka in den Properties einstelle, ich bekomme einfach nicht den original Wert angezeigt.
Hat jemand einen Tipp?

**Comment by Videonisse** [comment, marked as answer]
Have you tested with the Eve app or another 3rd party app? I think your issue is only with Apple Home app.

**Reply by JohnWayne007** [reply]
Thanks for your answer. Indeed it  is the Apple home App internal rounding. In Eve ist shows  the real values.
